cmd/logcli/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.13 as build
```

Comment:
could you change this to match the go version in the other docker files? 1.14.2?

Reply:
sure